modules/src/ics02_client/handler/update_client.rs

```diff
@@ -50,11 +50,13 @@ pub fn process(
 
     output.emit(ClientEvent::ClientUpdated(client_id.clone()));
 
-    Ok(output.with_result(UpdateClientResult {
-        client_id,
-        client_state: new_client_state,
-        consensus_state: new_consensus_state,
-    }))
+    Ok(
+        output.with_result(ClientResult::UpdateResult(UpdateClientResult {
```

Comment:
same here, `result` appears 4 times in this line, maybe we could reduce to:
```suggestion
        output.with_result(ClientResult::Update(Result {
```

Reply:
Not sure if perfect:

https://github.com/informalsystems/ibc-rs/commit/c1f8a4f4db0b9882c71cf1a6d63888ddc84437dd

Looking forward to hear your thoughts.